### ISSUE TITLE
Data split

### ISSUE THREAD

**Issue by stbedoya** [CSAILVision/GazeCapture#50]
Hi, 

You guys mentioned the the data is split by patient. Where can I get the patients ids used in train/validation/test set?

**Comment by erkil1452** [comment]
I assume patient = user/subject. In the dataset there are info.json files for each subject and it contains a field "Dataset" which is either 'train', 'val' or 'test'.

Alternatively, you can download the processed file
`https://github.com/CSAILVision/GazeCapture/blob/master/pytorch/reference_metadata.mat`
where there are binary masks `labelTest`, `labelVal`, `labelTrain` marking association of each image to a given split.
